src/transformers/training_args.py

```diff
@@ -1530,6 +1530,13 @@ class TrainingArguments:
         },
     )
 
+    average_tokens_across_devices: Optional[bool] = field(
+        default=False,
+        metadata={
+            "help": "Whether or not to average tokens across devices."
```

Comment:
Maybe we could share a bit more why this arg could be useful ?

Reply:
Done, please review my code.